utils/eventtracking.js

```diff
@@ -82,10 +85,18 @@ const track = async (eventType, eventProperties, options) => {
     }
 };
 
+async function getUserCountry() {
+    return fetch('https://ipinfo.io/json').then(
+        (response) => response.json()
+    ).then(
+        (jsonResponse) => jsonResponse.country
+    ).catch(() => console.log('Failed to get the country'));
```

Comment:
how about we also format the error just so we don't silently ignore it:
```suggestion
    ).catch(error => console.log(`Failed to get the country due to: ${error}`));
```